stripe.go

```diff
@@ -283,6 +297,8 @@ func (s *BackendConfiguration) NewRequest(method, path, key, contentType string,
 			}
 
 			req.Header.Add("Idempotency-Key", idempotency)
+		} else if method == http.MethodPost || method == http.MethodPut || method == http.MethodPatch || method == http.MethodDelete {
```

Comment:
Thanks for making this change!

Now that I look at this again, would you mind creating a new function called `isHTTPWriteMethod` (or such) and abstract this line into it? I think it'll be cleaner code-wise and more suggestive of why this check is being made.